### ISSUE TITLE
Inquiry Regarding Learning Rate and Training Epochs

### ISSUE THREAD

**Issue by ChiChivas** [nku-zhichengzhang/WECL#4]
Hello,
I am thoroughly reviewing the details mentioned in your paper, "Weakly Supervised Video Emotion Detection and Prediction via Cross-Modal Temporal Erasing Network," and attempting to replicate your experimental results from your open-source GitHub repository. In the paper, you mentioned: "We train the parameter of the attention module and linear classifier by a learning rate of 2e-4."

I have noticed that you set the default epoch to 100 in the code, but I have some questions regarding the adjustment of the learning rate during the training process and the number of training epochs to achieve the final results on the VideoEmotion-8 dataset. Specifically:
1. Did the learning rate of 2e-4 remain constant throughout the entire 100 epochs of training, as mentioned in the paper?
2. Were there any adjustments made to the learning rate during the experiment? If so, how were these adjustments made?
3. Could you provide details on the learning rate and the number of training epochs used to obtain the final experimental results on the VideoEmotion-8 dataset?

I did not find specific information on these details in the paper, and I hope to gain insights into these aspects to better understand your work and achieve similar results in my experiments.

Thank you!

**Comment by tangzhenwei2002** [comment]
Hello,
Can you help me run the code？
1.I see that the code has no requirements and don't know how to know which packages are needed to run the code?
Thank you!

**Comment by nku-zhichengzhang** [comment]
> Hello, I am thoroughly reviewing the details mentioned in your paper, "Weakly Supervised Video Emotion Detection and Prediction via Cross-Modal Temporal Erasing Network," and attempting to replicate your experimental results from your open-source GitHub repository. In the paper, you mentioned: "We train the parameter of the attention module and linear classifier by a learning rate of 2e-4."
> 
> I have noticed that you set the default epoch to 100 in the code, but I have some questions regarding the adjustment of the learning rate during the training process and the number of training epochs to achieve the final results on the VideoEmotion-8 dataset. Specifically:
> 
> 1. Did the learning rate of 2e-4 remain constant throughout the entire 100 epochs of training, as mentioned in the paper?
> 2. Were there any adjustments made to the learning rate during the experiment? If so, how were these adjustments made?
> 3. Could you provide details on the learning rate and the number of training epochs used to obtain the final experimental results on the VideoEmotion-8 dataset?
> 
> I did not find specific information on these details in the paper, and I hope to gain insights into these aspects to better understand your work and achieve similar results in my experiments.
> 
> Thank you!

Thanks for your interest.
1&2. Yes, we use constant lr as mentioned in our paper.
3. We train the model by 100 epochs with a lr of 2e-4.
